README.md

```diff
@@ -35,7 +35,7 @@ load("//tools/workspace/bazel_deps:repository.bzl", "bazel_deps_repository")
 
 def add_default_repositories(excludes = []):
     if "bazel_deps" not in excludes:
-        bazel_deps_repository(name = "bazel_deps")
+        bazel_deps_repository()
```

Comment:
I think using name="com_github_mjbots_bazel_deps" makes sense but hard-coding is OK

Reply:
Well, I stopped hard-coding it because things internal to the repo current require that name to be used.  Presenting an option that was guaranteed to never work seemed not friendly.